UI/window-basic-main.cpp

```diff
@@ -1676,7 +1676,13 @@ void OBSBasic::OBSInit()
 #endif
 
 	obs_data_t *obsData = obs_get_private_data();
+
+#ifdef _WIN32
 	vcamEnabled = obs_data_get_bool(obsData, "vcamEnabled");
+#elif __linux__
+	vcamEnabled = true;
```

Comment:
So, does this mean we're making v4l2loopback a hard dependency?

Reply:
Until I a find a way to check if its installed.

Reply:
Done.
https://github.com/obsproject/obs-studio/pull/3182/files#diff-fa2d26642c04410738a3dbd64db8b951R34